src/core/filters/meta/bpf/include/meta_filter.h

```diff
@@ -103,6 +108,9 @@ static __always_inline long meta_process_ops(struct retis_meta_ctx *ctx)
 			    !ptr)
 				return -1;
 
+			if ((void *)ptr == NULL)
+				return 0;
```

Comment:
Nit: I think this could be in a dedicated commit with an explanation about what this fixes.

Reply:
ah, right. Forgot to split this.

Reply:
looking at this twice, this is a leftover and should be dropped.
It was intended to be a replacement/slightly different approach for the previous `|| !ptr)`  (part of "bpf logger" and written/updated in parallel with this PR). 
This small redundant check fell through the cracks once eBPF logger got merged. Thanks for spotting this, though.